src/main/java/com/minecolonies/client/gui/WindowTownHall.java

```diff
@@ -314,11 +480,94 @@ public void updateElement(int index, Pane rowPane)
                 rowPane.findPaneOfTypeByID("name", Label.class).setLabelText(citizen.getName());
             }
         });
+    }
+
+    /**
+     * Fills the workOrder list inside the townhall GUI.
+     */
+    private void fillWorkOrderList()
+    {
+        ScrollingList workOrderList =  findPaneOfTypeByID(LIST_WORKORDER, ScrollingList.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'workOrderList' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)